src/scss/tools/mixins/_ratings.scss

```diff
@@ -0,0 +1,37 @@
+//
+// ==========================================================================
+// Ratings variant mixin
+// Used in alert modifier classes to colorize the alert
+// ==========================================================================
```

Comment:
Would be good to include an example of the classes produced by this mixin.

Looks to be something like `.c-ratings--fill-02`, `.c-ratings--fill-05` etc, but good to know for certain if anyone needs to edit it down the line.

Also, would be good to explain why we are creating these classes using the mixin - which is to allow us to control the width of the star ratings using classnames?